schema/sql/api.sql

```diff
@@ -125,6 +126,23 @@ comment on column v1.ssh_public_key.uid is
 alter view v1."ssh_public_key" owner to api;
 grant select on table v1."ssh_public_key" to "api-anon";
 
+create view v1.gpg_public_key as
+  select
+      encode(fingerprint, 'hex') as fingerprint,
+      encode(key, 'ascii') as key,
```

Comment:
any reason we don't have this as an "ASCII armored" public key?

Reply:
Is your suggestion to change this to

```
encode(key, 'ascii') as ascii_armoured_public_key
```

Reply:
If it's an ASCII armored public key in the database, why would it be stored as a bytea and not text, I guess is my question. If it is stored as some raw kind of data, we need to do some additions on top of it to make it *actually* a GPG-acceptable "ASCII armored" key.

Reply:
use `pgcrypto.armor`....
but also, have that as a computed column, and keep the binary key here (without the `encode`)